### PR TITLE
Added Yousician and FamilySearch

### PR DESCRIPTION
**Pull request by FreetimeFM**
Added Yousician.com

### CODE REVIEW

**Comment by FreetimeFM** [comment]
Fixed error where Yousician wasn't in the correct order.

**Comment by FreetimeFM** [comment]
Added FamilySearch

File: _data/sites.json [hunk 1]
```diff
@@ -4597,6 +4597,16 @@
         ]
     },
 
+    {
+        "name": "FamilySearch",
+        "url": "https://www.familysearch.org/identity/settings/account",
+        "difficulty": "easy",
+        "notes": "Go into your account settings and then click \"Delete Account\" (at the bottom of the page).",
```
Comment (tupaschoal) [inline _data/sites.json:4604]:
```suggestion
        "notes": "Go into your account settings and then click \"Delete Account\" at the bottom of the page.",
```

File: _data/sites.json [hunk 2]
```diff
@@ -15968,6 +15978,16 @@
         ]
     },
 
+    {
+        "name": "Yousician",
+        "url": "https://account.yousician.com/",
+        "difficulty": "easy",
+        "notes": "Go to your account (scroll down) and click \"Delete account\" in the \"Advanced Settings\" section. You will be prompted to enter your password.",
```
Comment (tupaschoal) [inline _data/sites.json:15985]:
```suggestion
        "notes": "Go to your account, scroll down and click \"Delete account\" in the \"Advanced Settings\" section. You will be prompted to enter your password.",
```

**Pull request merged**